src/main/java/lotto/LottoPrize.java

```diff
@@ -1,33 +1,48 @@
 package lotto;
 
-import util.OutputView;
+import java.util.Arrays;
 
 public enum LottoPrize {
-  THREE(3, 5000, "3개 일치 (%s원)- %s개"),
-  FOUR(4, 50000, "4개 일치 (%s원)- %s개"),
-  FIVE(5, 1500000, "5개 일치 (%s원)- %s개"),
-  SIX(6, 2000000000, "6개 일치 (%s원)- %s개");
+  NONE(0, 0, false),
+  THREE(3, 5000, false),
+  FOUR(4, 5_0000, false),
+  FIVE(5, 150_0000, false),
+  FIVE_AND_BONUS(5, 3000_0000, true),
+  SIX(6, 20_0000_0000, false);
```

Comment:
보통 언더스코어는 3자리로 끊습니다. 
```suggestion
  THREE(3, 5000, false),
  FOUR(4, 50_000, false),
  FIVE(5, 1_500_000, false),
  FIVE_AND_BONUS(5, 30_000_000, true),
  SIX(6, 2_000_000_000, false);
```